### DISCUSSION TITLE
Tool Swapper & Enchantments

### DISCUSSION THREAD

**Discussion by maksemakes** [P3pp3rF1y/SophisticatedBackpacks#934] [Q&A]
Is there a way to set what tool is swapped based on the enchantments on the tool? Or by the name of the tool? I have a fortune pick that I want to swap in only when mining ores, or a silk touch axe when trying to pick up things like glass or beehives.

**Comment by P3pp3rF1y** [comment]
No there's no way where a type of block would get translated to a tool to be used. It is based on the speed at which the given tool can mine given block, picking the fastest for the job. And I also don't see how to make the configuration of the block to tool reasonable so that it wouldn't require every single block to be defined even if it's just for exceptions

**Comment by maksemakes** [comment]
Ah, I understand.  I'll just have to keep the (specific) enchanted tools out of the tool swapper.  If you ever do find a feasible way that could be a good upgrade. Thank you!
